apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/CodeHealthTest.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Google.Cloud.ClientTesting;
```

Comment:
Copyright notice required ;)

Reply:
Whoops, thanks. Good to see CI caught that too.